langchain/src/util/async_caller.ts

```diff
@@ -100,6 +104,27 @@ export class AsyncCaller {
     );
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  callWithOptions<A extends any[], T extends (...args: A) => Promise<any>>(
+    options: AsyncCallerCallOptions,
```

Comment:
Options second?

Reply:
No, because with of the rest args signature it can't come last, so must come first